esmvaltool/diag_scripts/monitor/multi_datasets.py

```diff
@@ -647,6 +647,10 @@
     fontsize plus 2. Does not affect suptitles. If not given, use default
     matplotlib values. For a more fine-grained definition of fontsizes, use the
     option ``matplotlib_rc_params`` (see above).
+legend_kwargs: dict, optional
+    Optional keyword arguments for :func:`matplotlib.pyplot.legend`,
+    e.g. ``loc``. A legend is only shown if legend_kwargs are given,
+    default is no legend.
```

Comment:
It looks like this is not used currently

Reply:
Yes, it is currently not used anymore. The idea to include it, was to compare one model with the CMIP6 ensemble and observations/reanalysis. At the moment it was said not to include the CMIP6 ensemble for REF, therefore the recipe does not use it. But I'm not sure if I should remove it, since I think it is useful to include legends in some cases?

Reply:
Removed